assignments/objects.js

```diff
@@ -19,27 +19,88 @@ const example = {
 
 // Write your intern objects here:
 
+const internMitzi = {
+
+"id": 1,
+"name": "Mitzi",
+"email": "mmelloy0@psu.edu",
+"gender": "F",
+}
+
+const internKennan = {
+
+"id": 2,
+"name": "Kennan",
+"email": "kdiben1@tinypic.com",
+"gender": "M",
+}
+
+const internKeven = {
+
+"id": 3,
+"name": "Keven",
+"email": "kmummery2@wikimedia.org",
+"gender": "M",
+}
+
+const internGannie = {
+
+"id": 4,
+"name": "Gannie",
+"email":"gmartinson3@illinois.edu",
+"gender": "M",
+}
+
+const internAntonietta ={
+
+"id": 5,
+"name": "Antonietta",
+"email": "adaine5@samsung.com",
+"gender": "F",
+}
+
 
 // ==== Challenge 2: Reading Object Data ==== 
 // Once your objects are created, log out the following requests from HR into the console:
 
 // Mitzi's name
 
+console.log(internMitzi.name);
+
 // Kennan's ID
 
+console.log(internKennan.id);
+
 // Keven's email
 
+console.log(internKeven.email);
+
 // Gannie's name
 
+console.log(internGannie.name);
+
 // Antonietta's Gender
 
+console.log(internAntonietta.gender);
+
 // ==== Challenge 3: Object Methods ==== 
 // Give Kennan the ability to say "Hello, my name is Kennan!" Use the console.log provided as a hint.
 // console.log(kennan.speak());
 
```

Comment:
This works. We can also make this a method of the internKennon object.
"internKennan.speak = function() { .... }"